src/app/login/LoginForm.tsx

```diff
@@ -0,0 +1,166 @@
+"use client";
+
+import {
+  Box,
+  Button,
+  Typography,
+  Avatar,
+  Dialog,
+  DialogContent,
+  DialogContentText,
+  DialogActions,
+  Snackbar,
+} from "@mui/material";
+import { useForm, FormProvider } from "react-hook-form";
+import { login } from "@/apis/auth";
+import SenifitTextField from "../../components/SenifitTextField";
+import { useRouter } from "next/navigation";
+import { useState } from "react";
+
+type LoginFormValues = {
+  id: string;
+  password: string;
+};
+
+export default function LoginForm() {
+  const methods = useForm<LoginFormValues>({ mode: "onBlur" });
+  const { handleSubmit, control, formState: { errors } } = methods;
+
+  const router = useRouter();
+  const [showSnackbar, setShowSnackbar] = useState(false);
+  const [showDialog, setShowDialog] = useState(false);
+
+  const onSubmit = async (data: LoginFormValues) => {
+    try {
+      await login(data);
+      setShowSnackbar(true);
+      setTimeout(() => {
+        router.push("/exercise");
+      }, 1000);
+    } catch (error) {
+      setShowDialog(true);
+    }
+  };
+
+  return (
+    <FormProvider {...methods}>
```

Comment:
FormProvider가 아닌 `form`태그 사용 부탁드립니다.

```suggestion
    <form onSubmit={handleSubmit(onSubmit)}>
```